coderd/coderd.go

```diff
@@ -1494,6 +1494,10 @@ func New(options *Options) *API {
 			r.Use(apiKeyMiddleware)
 			r.Get("/", api.tailnetRPCConn)
 		})
+		r.Route("/aitasks", func(r chi.Router) {
```

Comment:
Blocker: we need to finalize whether this will be a v2 endpoint or experimental.